src/startup/mldr/mldr.c

```diff
@@ -188,6 +188,8 @@ int main(int argc, char** argv, char** envp)
 		exit(1);
 	}
 
+	dserver_rpc_set_executable_path(filename, strlen(filename));
```

Comment:
Error handling, just in case (which is just exiting since we can't actually do anything without the server):
```suggestion
	if (dserver_rpc_set_executable_path(filename, strlen(filename)) < 0) {
		fprintf(stderr, "Failed to tell darlingserver about our executable path\n");
		exit(1);
	}
```